src/questions/install-command.js

```diff
@@ -2,5 +2,12 @@ module.exports = projectInfos => ({
   type: 'input',
   message: '📦  Install command (use empty value to skip)',
   name: 'installCommand',
-  default: projectInfos.isJSProject ? 'npm install' : undefined
+  default: answers => {
+    if (projectInfos.isJSProject) {
+      const packageManager =
+        projectInfos.packageManager || answers.packageManager
+      return `${packageManager} install`
+    }
+    return undefined
```

Comment:
Now, it could just be :

```
default: answers => {
const packageManager = answers.packageManager || projectInfos.packageManager
return isNil(packageManager) ? undefined : `${packageManager} install`
}
```